modules/mobile/app/models/mobile/v0/vaccine.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    # Stores vaccine data from the CDC for use in Immunization records
+    # @example create a new instance
+    #   Mobile::V0::Vaccine.create(cvx_code: 1, group_name: 'FLU', manufacturer: 'Moderna')
+    #
+    class Vaccine < ApplicationRecord
+      validates :cvx_code, presence: true, uniqueness: true
+      validates :group_name, presence: true
+
+      # Adds incoming name to group name unless it's already included
+      # does not save record
```

Comment:
The fact that this doesn't update the record is a bit atypical, but it helps us to prevent saving twice if we also need to update the manufacturer.